07_errors/alextmz/errors.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import "fmt"
+
+type Error struct {
+	Message string
+	Code    uint16
+}
+
+const (
+	_ = iota
+	ErrGeneric
+	_
+	ErrValueLessThanZero
+	_
+	ErrIDBeingCreatedAlreadyExists
```

Comment:
nit: IMO error codes can be simplified.
eg:
```
ErrInvalid   
ErrDoesNotExist
ErrInternal
```

Reply:
Agree, reading Lab 10 makes more sense to use HTTP-like codes.